source/install/cloud-changelog.md

```diff
@@ -4,12 +4,73 @@ This changelog summarizes updates to [Mattermost Cloud](https://mattermost.com/g
 
 Latest Mattermost Cloud releases:
 
+- [Release 2022-06-15](#release-2022-06-15)
 - [Release 2022-05-26](#release-2022-05-26)
 - [Release 2022-05-12](#release-2022-05-12)
 - [Release 2022-04-28](#release-2022-04-28)
 - [Release 2022-04-13](#release-2022-04-13)
 - [Release 2022-03-30](#release-2022-03-30)
-- [Release 2022-03-16](#release-2022-03-16)
+
+## Release 2022-06-15
+
+### Highlights
+
+#### Free Forever Mattermost Cloud Plan
+ - Mattermost Cloud now supports a free forever plan, Cloud Starter, for unlimited users for unlimited time.
+ - Access unlimited channels, playbooks, and boards across the Mattermost platform, with unlimited voice calls and screen sharing in Direct Messages.
+ - The free plan includes a few workspace limits. [Upgrade to Professional for unlimited access](https://mattermost.com/pricing/):
+    - Ability to start a free 30-day Enterprise trial, with access to all features during the trial period.
+    - Maximum 1 team. Additional teams can be created during a trial or on a paid plan, which are archived if downgraded back to Starter. History will be preserved, though that archived history can only be accessed if a free user upgrades to a paid subscription plan.
+    - 10GB file storage across the platform, with 100MB upload limit. History will be preserved.
+    - Unlimited installed apps or plugins, with maximum of 5 enabled at one time. History will be preserved.
+    - Access to 10,000 most recent messages. History will be preserved.
+    - Access to 500 most recently updated cards.
+    - Maximum 5 saved views per board.
+
+#### Calls (Beta)
+ - Native voice calling and screen sharing is now available. This is a Channels-specific integration.
+
+#### Apps Bar (Beta)
+ - The channel header is now decluttered to make it more obvious how to access Calls, Playbooks, and Boards when viewing a channel. All channel header icons registered by plugins are moved to the new Apps Bar, while Calls remains in the channel header.
+
+#### Collapsed Reply Threads (General Availability)
+ - [Collapsed Reply Threads](XXXXXX link to CRT GA blog post) is now generally available. Please reference [this article](link to KB article XXXXXX) for more information.
+
+### Improvements
+
+#### User Interface (UI)
+ - Applied new designs for the Login screen.
+ - Changed some tooltips to appear when focused instead of just on hover.
+ - The legacy **Settings > Advanced Settings > Enable Post Formatting** and **Settings > Advanced Settings > Preview Pre-release Features** settings are now deprecated in favor of the new message formatting toolbar.
+ - Romanian language support was downgraded to Beta.
+
+#### Administration
+ - For new Cloud Workspaces, the default value for **Collapsed Reply Threads** in **System Console > Experimental > Features** is now set to **Always On**. Does not impact existing workspaces where this value is already set.
```

Comment:
We're moving this config setting from Experimental to Site Configuration > Posts via https://github.com/mattermost/mattermost-webapp/pull/10465